src/v/pandaproxy/schema_registry/configuration.cc

```diff
@@ -32,7 +32,7 @@ configuration::configuration()
       {},
       {},
       config::endpoint_tls_config::validate_many)
-  , mode_mutability(*this, "mode_mutability", "Allow modifying mode", {}, false)
+  , mode_mutability(*this, "mode_mutability", "Allow modifying mode", {}, true)
```

Comment:
We don't do this for node config as it currently isn't exported, but the properties automation script might benefit from it:
```suggestion
  , mode_mutability(
      *this,
      "mode_mutability",
      "Allow modifying mode",
      {.visibility = config::visibility::tunable},
      true)
```